app/src/main/java/ru/qveex/rst_tur/presentation/components/lists/FoodList.kt

```diff
@@ -39,7 +38,7 @@ fun FoodList(foods: List<Fun>) {
         }
         Box(modifier = Modifier.padding(vertical = 20.dp)) {
             RoundedButton(
-                text = "Показать все (${foods.size})",
+                text = stringResource(R.string.show_all_food_items, foods.size),
```

Comment:
```suggestion
                text = stringResource(R.string.show_all, foods.size),
```
You can rename this string res
Because we can reuse this string in any other cases to show all items of any list